lblod/app-lokaal-mandatenbeheer
# Ben/lmb 575 add installatievergadering data

## Description

Temporary migration to add installatievergaderingen concepten for all relevant bestuurseenheden.

## How to test

Run the migration and see if all gemeente bestuurseenheden now have an installatievergadering page. Be aware this is a migration that adds these installatievergaderingen for the period 2019-2025, since not all bestuursorganen already have a new bestuursorgaan in de tijd for the next bestuursperiod. This will be updated once we have the new bestuursorganen.
